client/src/AppTest.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
What is the rationale behind the naming convention of this file? It sounds more like a test file than a normal component app